### ISSUE TITLE
"Failed running AppleScript" on macOS 15 Sequoia

### ISSUE THREAD

**Issue by ncosgray** [create-dmg/create-dmg#186]
I've been using the same create-dmg script for years and it was working nicely prior to upgrading to Sequoia. Sadly, now my script is erroring out with "Failed running AppleScript" at what appears to be the prettification step. Platform is macOS 15 Sequoia running on M1 Pro. The error occurs on the latest 1.2.2 release as well as on 1.1.0 (what I was using before today).

I also get this error when I try to run the **[sample](https://github.com/create-dmg/create-dmg/blob/master/examples/01-main-example/sample)** script, so here's the output from **sample** for reference:
```
01-main-example % ./sample                  
Creating disk image...
...
created: rw.59634.Application-Installer.dmg
Mounting disk image...
Device name:     /dev/disk10
Searching for mounted interstitial disk image using /dev/disk10s... 
Mount dir:       /Volumes/dmg.VmNqP4
Copying background file 'installer_background.png'...
Making link to Applications dir...
/Volumes/dmg.VmNqP4
Running AppleScript to make Finder stuff pretty: /usr/bin/osascript "/var/folders/1g/_9gflqfj0hj32_1j21shg1t40000gn/T/createdmg.tmp.XXXXXXXXXX.rPXzQlxLHi" "dmg.VmNqP4"
/var/folders/1g/_9gflqfj0hj32_1j21shg1t40000gn/T/createdmg.tmp.XXXXXXXXXX.rPXzQlxLHi:85:89: execution error: Finder got an error: Can’t get disk "dmg.VmNqP4". (-1728)
Failed running AppleScript
Unmounting disk image...
"disk10" ejected.
```

Thanks for this useful tool, and thanks for taking a look at my issue! :)

**Comment by pzhlkj6612** [comment]
Hi. Not a maintainer but please allow me to add two links.

<br>

This is a new error about "Finder got an error: Can’t get disk .... (-1728)":

- The issue on 20160811: https://github.com/create-dmg/create-dmg/issues/43
- The fix on 20180927: https://github.com/create-dmg/create-dmg/pull/55

.

**Comment by ncosgray** [comment]
After installing the macOS 15.1 update, this issue has gone away.
